docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters.md

```diff
@@ -36,56 +36,69 @@ Standard clusters include the following features:
 
 ## Prerequisites
 
-- Create a billing profile and add a payment method before you can create a standard cluster. Refer to [Manage your billing profile and payment method](../../cloud-admin/cloud-billing-profile/).
+- Create a billing profile and add a payment method before you can create a Dedicated cluster. Refer to [Manage your billing profile and payment method](../../cloud-admin/cloud-billing-profile/).
 - If you want to use dedicated VPCs for network isolation and security, create the VPC before you create your cluster. YugabyteDB Managed supports AWC and GCP for peering. Refer to [VPC networking](../../cloud-basics/cloud-vpcs/).
 
 ## Create a cluster
 
 To create a cluster, on the **Clusters** page, click **Add Cluster** to start the **Create Cluster** wizard.
 
-The **Create Cluster** wizard has the following three pages:
+The **Create Cluster** wizard has the following pages:
 
 1. [Select Cluster Type](#select-cluster-type)
-1. [Cluster Settings](#cluster-settings)
-1. [Database Admin Credentials](#database-admin-credentials)
+1. [General Settings](#general-settings)
+1. [Cluster Setup](#cluster-setup)
+1. [DB Credentials](#db-credentials)
 
 ### Select Cluster Type
 
 ![Add Cluster Wizard - Select Type](/images/yb-cloud/cloud-addcluster-paid1.png)
 
-Select **YugabyteDB Managed** and click **Next** to display the **Cluster Settings** page.
+Select **Dedicated** and click **Choose** to display the **General Settings** page.
 
-### Cluster Settings
+### General Settings
 
-![Add Cluster Wizard - Cluster Settings](/images/yb-cloud/cloud-addcluster-paid2.png)
+![Add Cluster Wizard - General Settings](/images/yb-cloud/cloud-addcluster-free2.png)
 
 Set the following options:
 
-- **Provider**: Choose a cloud provider - AWS or GCP. (For Azure, contact {{<support-cloud>}}.)
 - **Cluster Name**: Enter a name for the cluster.
-- **Region**: Choose the [region](../../release-notes#cloud-provider-regions) where the cluster will be located, or click **Request a multi-region cluster** to contact Yugabyte Support to arrange multi-region deployment.
+- **Provider**: Choose a cloud provider - AWS or GCP. (For Azure, contact {{<support-cloud>}}.)
```

Comment:
Suggestion:

Choose a cloud provider - AWS or GCP. (For Azure, contact {{<support-cloud>}}.)  >

Choose a cloud provider: AWS or GCP. If you want to use Azure, contact {{<support-cloud>}}.